tests/approaches/test_grammar_search_invention_approach.py

```diff
@@ -209,7 +209,13 @@ def test_create_score_function():
                                         {})
     assert isinstance(score_func,
                       _RelaxationHeuristicLookaheadBasedScoreFunction)
-    assert score_func.heuristic_name == "hff"
+    assert score_func.heuristic_names == ["hff"]
+    score_func = _create_score_function("lmcut,hff_lookahead_depth0", set(),
```

Comment:
it's a bit sad that my brain parses `_` as taking precedence over `,` due to normal coding patterns, but oh well